plugins/git-commit/git-commit.plugin.zsh

```diff
@@ -14,6 +14,56 @@ _git_commit_aliases=(
   'wip'
 )
 
+_git_commit_flags=(
+  '-s'
+  '--scope'
+  '-a'
+  '--attention'
+  '-sa'
+  '-as'
+)
+
+remove_flags() {
+  local regex=$(IFS="|"; echo "${_git_commit_flags[*]}")
```

Comment:
There's a better way to do it without spawning a subshell!
```suggestion
  local regex="${(j:|:)_git_commit_flags}"
```

Reply:
Interesting, I liked that!

Reply:
Indeed, it's a `zsh` syntax thing

Reply:
Got it, cool!